### PR TITLE
created list of categories components and added to side bar

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: directory-of-services/src/Components/Categories/Categories.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { List, ListItem } from 'material-ui/List';
+import { Link } from "react-router-dom";
+import './categories.css'
+
+const styles = {
```
Comment (kkarimi) [inline directory-of-services/src/Components/Categories/Categories.js]: Let's stick to a consistent method, either `import css...` or `style = {}`, in here I think for now it is best to stick to a separate CSS file.

File: directory-of-services/src/Components/Categories/Categories.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { List, ListItem } from 'material-ui/List';
+import { Link } from "react-router-dom";
+import './categories.css'
+
```
Comment (kkarimi) [inline directory-of-services/src/Components/Categories/Categories.js]: Please add the categories from the screenshot?

File: directory-of-services/src/Components/Categories/Categories.js (hunk 1 above)
Comment (DwinaTech) [reply directory-of-services/src/Components/Categories/Categories.js]: I am using const style here only to add active class by updating the state, is there better way to add active class in react?

Comment (kkarimi) [reply directory-of-services/src/Components/Categories/Categories.js]: That's a good call @DwinaTech, though I guess we don't really need this functionality yet

File: directory-of-services/src/Components/Categories/Categories.js (hunk 2 above)
Comment (DwinaTech) [reply directory-of-services/src/Components/Categories/Categories.js]: Done

File: directory-of-services/src/Components/Categories/Categories.js (hunk 1 above)
Comment (DwinaTech) [reply directory-of-services/src/Components/Categories/Categories.js]: alright

Comment (DwinaTech) [reply directory-of-services/src/Components/Categories/Categories.js]: Done

**Pull request merged**